api/types.go

```diff
@@ -503,6 +503,31 @@ type CreateRequest struct {
 	Quantization string `json:"quantization,omitempty"`
 }
 
+type Files []File
```

Comment:
Is this new type needed to fix the bug? From the brief look I've taken, it adds a `name` and `path` (the original key of the `map[string]string`), but `name` could always be found by calling `filepath.Base(path)` anyways